docs-src/content/connecting/_index.md

```diff
@@ -46,22 +46,27 @@ func main() {
 {{< /tab >}}
 
 {{< tab "C++" "c++" >}}
-// File - main.cpp
+#include <diatheke_client.h>
+#include <iostream>
 
-#include "diatheke_client.h"
+int main(int argc, char* argv[])
+{
+  std::string serverAddress = "127.0.0.1:9002";
 
-#include <iostream>
-#include <string>
+  // Read or embed the digital certificate, and store
+  // in rootCert. Unlike other languages, in C++ there
+  // is nothing to do this automatically.
+  grpc::string rootCert;
```

Comment:
should we have some notes here for where to read this from on linux/windows?

Reply:
Well, TBH, I'm not sure where to read it from on Linux. On Windows, you have to use the Windows API and get it from the [certificate store](https://docs.microsoft.com/en-us/windows/win32/seccrypto/using-certificates). And what about Mac? Do we want to document where to get it from for all major platforms?